learing_journal/tests.py

```diff
@@ -42,23 +43,23 @@ def teardown():
 
 
 def test_model_gets_added(new_session):
-    assert len(new_session.query(MyModel).all()) == 0
-    model = MyModel(title="test", body='some text')
+    assert len(new_session.query(Journal).all()) == 0
+    model = Journal(title="test", body='some text')
     new_session.add(model)
     new_session.flush()
-    assert len(new_session.query(MyModel).all()) == 1
+    assert len(new_session.query(Journal).all()) == 1
 
 
 def test_model_gets_added_2(new_session):
     """Test the creation of 2 new models in the row."""
-    assert len(new_session.query(MyModel).all()) == 0
-    model = MyModel(title="test_day", body="test_body")
+    assert len(new_session.query(Journal).all()) == 0
+    model = Journal(title="test_day", body="test_body")
     new_session.add(model)
     new_session.flush()
-    model = MyModel(title="test_day2", body="test_body2")
+    model = Journal(title="test_day2", body="test_body2")
     new_session.add(model)
     new_session.flush()
```

Comment:
The above six lines are really three lines repeated twice.  Can you avoid that kind of repetition?